### PR TITLE
BIP174 nits

### PR DESCRIPTION
**Pull request by knocte**
Some nitpicks fixed in the recently committed BIP174 code.

### CODE REVIEW

**Comment by joemphilips** [comment]
Ah, thanks! looks nice for except one thing.

**Comment by knocte** [comment]
What thing? the build? I know, I just pushed the fix for it :P

File: NBitcoin/BIP174/PartiallySignedTransaction.cs [hunk 1]
```diff
@@ -1517,12 +1506,12 @@ public PSBT CoinJoin(PSBT other)
 		/// </summary>
 		/// <param name="errors"></param>
 		/// <returns></returns>
-		public PSBT Finalize(out InvalidOperationException[] errors)
+		private PSBT Finalize(out InvalidOperationException[] errors)
```
Comment (joemphilips) [inline NBitcoin/BIP174/PartiallySignedTransaction.cs]: I was expecting this to be a public first, but it may not be a big difference

**Comment by joemphilips** [comment]
Nevermind, I've just commented, I changed my mind that it seems not a big deal.

File: NBitcoin/BIP174/PartiallySignedTransaction.cs (hunk 1 above)
Comment (knocte) [reply NBitcoin/BIP174/PartiallySignedTransaction.cs]: but the public overload still throws an aggregateException, in .NET this is the way to get error codes (except if you use the Try pattern, in which case it doesn't throw, to save performance)

**Comment by joemphilips** [comment]
LGTM except for the build.

**Comment by knocte** [comment]
Seems one test failed the CI because of QBitNinja?:

```
Starting test execution, please wait...
dotnet : [xUnit.net 00:06:04.00]     NBitcoin.Tests.RepositoryTests.CanRequestTransactionOnQBit [FAIL]
At line:3 char:1
+ dotnet test -c Release ./NBitcoin.Tests/NBitcoin.Tests.csproj --filte ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: ([xUnit.net 00:0...onOnQBit [FAIL]:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
Failed   NBitcoin.Tests.RepositoryTests.CanRequestTransactionOnQBit
Error Message:
 System.Net.Http.HttpRequestException : Response status code does not indicate success: 502 (Bad Gateway).
Stack Trace:
   at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at NBitcoin.QBitNinjaTransactionRepository.<GetAsync>d__9.MoveNext() in C:\projects\nbitcoin\NBitcoin\QBitNinjaTransactionRepository.cs:line 72
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at NBitcoin.TxRepoExtensions.Get(ITransactionRepository repo, uint256 txId) in C:\projects\nbitcoin\NBitcoin\ITransactionRepository.cs:line 40
   at NBitcoin.Tests.RepositoryTests.CanRequestTransactionOnQBit() in C:\projects\nbitcoin\NBitcoin.Tests\RepositoryTests.cs:line 80
Total tests: 350. Passed: 349. Failed: 1. Skipped: 0.
Test Run Failed.
```

@NicolasDorier you need to restart it?

**Comment by NicolasDorier** [comment]
yeah sorry for that!

**Pull request merged**